rental.rb

```diff
@@ -1,7 +1,7 @@
 class Rental
   attr_accessor :date, :book, :person
 
-  def initialize(date)
+  def initialize(book, person, date)
```

Comment:
- [ ] Please fix the error when creating rental.

![image](https://user-images.githubusercontent.com/38635158/164909153-cf1339b9-fed5-44fe-8587-4c3aae5e815d.png)